### ISSUE TITLE
Riferimento ai dati DPC

### ISSUE THREAD

**Issue by iltempe** [emergenzeHack/covid19italia#38]
Inserire link e riferimenti agli opendata della protezione civile.

**Comment by cristigalas** [comment]
Se li inseriamo qui? #30 
Oppure facciamo una sezione sul sito dedicata solo ai dati ufficiali? Forse sarebbe meglio...

**Comment by iltempe** [comment]
@aborruso, io iniziarlmente linkerei quelli del governo in una pagina di link utili.

**Comment by tailot** [comment]
@aborruso https://github.com/pcm-dpc/COVID-19

**Comment by iltempe** [comment]
ok esiste una pagina "opendata.md" che è popolata dal file dati opendata.csv compilando il CSV a mano con le info relative ai dati automaticamente nella pagina si popola la tabella. E poi mettiamo un link in home page. qualcuno se lo assegna?

**Comment by cristigalas** [comment]
@iltempe secondo me fare questa pagina, come suggerivi, sarebbe importante e urgente....

**Comment by iltempe** [comment]
@cristigalas ci sono già nelle fonti istituzionali? almeno chiudo.

**Comment by cristigalas** [comment]
> @cristigalas ci sono già nelle fonti istituzionali? almeno chiudo.

No, non ci sono e quella pagina è ancora tutta da sistemare, compresa la visualizzazione :(

**Comment by cristigalas** [comment]
A proposito di una sezione Dati e fase2 (vedi #560) in posta ci scrive un giovane programmatore che ha aperto questo sito https://www.curaitalia.it/ che rielabora a livello nazionale e regionale i dati sul coronavirus. Semmai potremmo chiedergli se ha voglia di collaborare con noi e aiutarci a realizzare la pagina Dati sul nostro sito... 
@chiccap @iltempe

**Comment by chiccap** [comment]
Ma che bello! Mi sembra un’ottima occasione! @iltempe @cristigalas

**Comment by RiccardoBorchi** [comment]
> A proposito di una sezione Dati e fase2 (vedi #560) in posta ci scrive un giovane programmatore che ha aperto questo sito https://www.curaitalia.it/ che rielabora a livello nazionale e regionale i dati sul coronavirus. Semmai potremmo chiedergli se ha voglia di collaborare con noi e aiutarci a realizzare la pagina Dati sul nostro sito...
> @chiccap @iltempe

Ciao! Ho visto adesso questo messaggio, per caso. Sono il creatore di www.curaitalia.it. Se siete ancora interessati a una eventuale collaborazione con il vostro sito fatemi sapere! :)

**Comment by RiccardoBorchi** [comment]
@cristigalas @chiccap @iltempe 

Fatemi sapere se foste ancora interessati a collaborare con il mio progetto! :)
